src/software/power/control_executor.cpp

```diff
@@ -72,12 +69,11 @@ void ControlExecutor::execute(const TbotsProto_PowerControl& control)
                     charger->chargeCapacitors();
```

Comment:
Lets remove ChargeControl entirely from PowerControl proto. And call this in setup()